src/tools/compiletest/src/runtest/debuginfo.rs

```diff
@@ -59,14 +59,8 @@ impl TestCx<'_> {
             return;
         }
 
-        let prefixes = {
-            static PREFIXES: &[&str] = &["cdb", "cdbg"];
-            // No "native rust support" variation for CDB yet.
-            PREFIXES
-        };
-
         // Parse debugger commands etc from test files
-        let dbg_cmds = DebuggerCommands::parse_from(&self.testpaths.file, self.config, prefixes)
+        let dbg_cmds = DebuggerCommands::parse_from(&self.testpaths.file, self.config, "cdb")
```

Comment:
I did some digging, this was introduced in the very original PR that added cdb support, i.e. https://github.com/rust-lang/rust/pull/60970. There was no discussion about `cdbg` in the PR either.

Possibilities:

- This refers to some project that never really materialized: https://handmade.network/forums/wip/t/3017-windows_debugger
- This is actually referring to cgdb, https://cgdb.github.io

In any case, none of the debuginfo tests use it.

Reply:
At that time, gdb had the prefixes [`gdb`, `gdbr`, `gdbg`], and similarly for lldb.

I think the idea was that `r` would only apply when using a Rust-aware debugger, and `g` would only apply when using a non-Rust-aware debugger.

I imagine that the PR author copied what the other debuggers were doing, but without the `r` variant as there was no Rust-aware cdb.

Later, the whole r/g distinction was removed from gdb/lldb by #129218. But cdb was not updated, so the useless g remained.